server/go.mod

```diff
@@ -204,6 +204,7 @@ require (
 	github.com/splitio/go-toolkit/v5 v5.4.0 // indirect
 	github.com/ssor/bom v0.0.0-20170718123548-6386211fdfcf // indirect
 	github.com/stretchr/objx v0.5.1 // indirect
+	github.com/strukturag/libheif v1.17.6
```

Comment:
Unfortunately, we actually can't use this library because it's licensed GPL. We'll need to find one that's something more permissive like MIT or Apache licensed

Reply:
Good catch, I missed the license.

Reply:
how about https://github.com/jdeng/goheif ?

Reply:
This doesn't look like a library endorsed/used by the community. There are no releases, no code updates the last 4 years and the forks/stars indicate that it's not a library widely used.

Reply:
It also doesn't look like that has a license either which means it's also not an option.

I remember looking up libraries for this in the past, and I was surprised by the lack of options back then. It seems like there's very little support for HEIC/HEIF because of the codec licensing fees, especially given that browsers don't seem to natively support the format either. I'm honestly not sure if this is something that's even possible for us to support